types/applicationForm.ts

```diff
@@ -115,3 +115,61 @@ export interface ApplicationFormPrefill {
     phone?: Phone
 }
 
+export interface ApplicationForm {
+    /**
+     * Identifier of the applicationForm resource.
+     */
+    id: string
+
+    /**
+     * Type of the applicationForm resource.
+     */
+    type: "applicationForm"
+
+    attributes: {
+        /**
+         * One of ChooseBusinessOrIndividual, TODO: fill in other possible stages. Application Form Stages.
+         */
+         stage: string
+
+        /**
+         * Organization name for the resource.
+         */
+        orgName: string
+
+        /**
+         * Url for the applicationForm resource.
+         */
+         url: string
+
+        /**
+         * Bank Name for the applicationForm resource.
+         */
+         bankName: string
+
+        /**
+         * Application form settings for the applicationForm resource.
+         * TODO: unsure of values, fill in properties
+         */
+        applicationFormSettings: object
+
+        /**
+         * Details for the application for the applicationForm resource.
+         */
+        applicantDetails: {
+            /**
+             * Type of applicant. TODO: unsure of values
+            */
+            applicationType: "Individual" | "Business"
+
+            /**
+             * Email address of the individual.
+            */
+            email: string
+        }
+        /**
+         * See [Tags](https://developers.unit.co/#tags).
+         */
+        tags: object
+    }
+}
```

Comment:
Hey @jerrywang-hiatus :)

Sorry for the long delay,
We are in the process of updating the docs regarding the application form, some of the properties that were previously returned have been omitted from the API call (they were more relevant to the actual UI).
The only properties that will be returned in the `create` and `get` endpoints will be: `tags, url, stage, applicantDetails` (You can see https://docs.unit.co/#create-application-form response section for more details).

Other than that everything looks great, and ApplicationForm as a resource will soon be updated in the docs (including relationship information)

Will tell you once it's live!

Thanks :)

Reply:
Hi again, here's the relevant new docs section about the ApplicationForm resource: https://docs.unit.co/#applicationform